docs/quickstart.rst

```diff
@@ -41,17 +46,18 @@ It is recommended to use a virtual environment when installing FEDn.
 
 .. _package-creation:
 
-Initialize FEDn with the client code bundle and seed model 
+Initialize FEDn with the compute package and seed model 
```

Comment:
We want to move away from calling it compute package and instead refer to is as a FEDn project which can be bundled into a package which then can be used by the fedn package manager.